hub/scripts.js

```diff
@@ -397,6 +397,39 @@
     });
   }
 
+  async function decoratePromotions() {
+    if (document.querySelector('main .promotion') instanceof HTMLElement) {
+      // A promotion has already been defined on the page.
+      // Do not inject another one.
+      return;
+    }
+
+    let promotionName = document.querySelector('head meta[name="promotion"]');
+    if (!promotionName) {
+      return;
+    }
+    promotionName = promotionName.getAttribute('content').toLowerCase();
```

Comment:
Will it always have a content attribute or should we add a small safeguard/move it into the try/catch?

```suggestion
    promotionName = (promotionName.getAttribute('content') || '').toLowerCase();
```

Reply:
it will always have a `content` attribute, yes; rendering is done from Helix